{
  "uid" : "bad55fc6c53e8aa4",
  "name" : "Verify clickability, href of links on the page",
  "fullName" : "tests.contributors_page_test.TestContributorsPageLinks#test_cnp_03_01_verify_links_on_page",
  "historyId" : "342402ff4b6c443b81ee3701714ca4e5",
  "time" : {
    "start" : 1728784124460,
    "stop" : 1728784124460,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55ec6566202a <unknown>\n#1 0x55ec653485e0 <unknown>\n#2 0x55ec653407eb <unknown>\n#3 0x55ec65330a09 <unknown>\n#4 0x55ec65332744 <unknown>\n#5 0x55ec65330cbd <unknown>\n#6 0x55ec6533052f <unknown>\n#7 0x55ec65330422 <unknown>\n#8 0x55ec6532e493 <unknown>\n#9 0x55ec6532eada <unknown>\n#10 0x55ec6534af41 <unknown>\n#11 0x55ec653dc8c5 <unknown>\n#12 0x55ec653bcb22 <unknown>\n#13 0x55ec653dbd7d <unknown>\n#14 0x55ec653bc8c3 <unknown>\n#15 0x55ec6538a6b3 <unknown>\n#16 0x55ec6538b68e <unknown>\n#17 0x55ec6562ca2b <unknown>\n#18 0x55ec656309b1 <unknown>\n#19 0x55ec65619225 <unknown>\n#20 0x55ec65631532 <unknown>\n#21 0x55ec655fe38f <unknown>\n#22 0x55ec65650f28 <unknown>\n#23 0x55ec656510f3 <unknown>\n#24 0x55ec65660e7c <unknown>\n#25 0x7f8c81894ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"8bca3ccebba11d3ebf0dc61fc7c53968\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_CONTRIBUTORS_PAGE}')\n    def contributors_page_open(driver):\n>       driver.get(MainPageLinks.URL_CONTRIBUTORS_PAGE)\n\ntests/conftest.py:28: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7faf1f7e8260>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55ec656510f3 \\\\u003Cunknown>\\\\n#24 0x55ec65660e7c \\\\u003Cunknown>\\\\n#25 0x7f8c81894ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x55ec6566202a <unknown>\nE       #1 0x55ec653485e0 <unknown>\nE       #2 0x55ec653407eb <unknown>\nE       #3 0x55ec65330a09 <unknown>\nE       #4 0x55ec65332744 <unknown>\nE       #5 0x55ec65330cbd <unknown>\nE       #6 0x55ec6533052f <unknown>\nE       #7 0x55ec65330422 <unknown>\nE       #8 0x55ec6532e493 <unknown>\nE       #9 0x55ec6532eada <unknown>\nE       #10 0x55ec6534af41 <unknown>\nE       #11 0x55ec653dc8c5 <unknown>\nE       #12 0x55ec653bcb22 <unknown>\nE       #13 0x55ec653dbd7d <unknown>\nE       #14 0x55ec653bc8c3 <unknown>\nE       #15 0x55ec6538a6b3 <unknown>\nE       #16 0x55ec6538b68e <unknown>\nE       #17 0x55ec6562ca2b <unknown>\nE       #18 0x55ec656309b1 <unknown>\nE       #19 0x55ec65619225 <unknown>\nE       #20 0x55ec65631532 <unknown>\nE       #21 0x55ec655fe38f <unknown>\nE       #22 0x55ec65650f28 <unknown>\nE       #23 0x55ec656510f3 <unknown>\nE       #24 0x55ec65660e7c <unknown>\nE       #25 0x7f8c81894ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1728784124460,
      "stop" : 1728784125230,
      "duration" : 770
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728784074935,
      "stop" : 1728784074936,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "contributors_page_open",
    "time" : {
      "start" : 1728784125230,
      "stop" : 1728784125658,
      "duration" : 428
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55ec6566202a <unknown>\n#1 0x55ec653485e0 <unknown>\n#2 0x55ec653407eb <unknown>\n#3 0x55ec65330a09 <unknown>\n#4 0x55ec65332744 <unknown>\n#5 0x55ec65330cbd <unknown>\n#6 0x55ec6533052f <unknown>\n#7 0x55ec65330422 <unknown>\n#8 0x55ec6532e493 <unknown>\n#9 0x55ec6532eada <unknown>\n#10 0x55ec6534af41 <unknown>\n#11 0x55ec653dc8c5 <unknown>\n#12 0x55ec653bcb22 <unknown>\n#13 0x55ec653dbd7d <unknown>\n#14 0x55ec653bc8c3 <unknown>\n#15 0x55ec6538a6b3 <unknown>\n#16 0x55ec6538b68e <unknown>\n#17 0x55ec6562ca2b <unknown>\n#18 0x55ec656309b1 <unknown>\n#19 0x55ec65619225 <unknown>\n#20 0x55ec65631532 <unknown>\n#21 0x55ec655fe38f <unknown>\n#22 0x55ec65650f28 <unknown>\n#23 0x55ec656510f3 <unknown>\n#24 0x55ec65660e7c <unknown>\n#25 0x7f8c81894ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 28, in contributors_page_open\n    driver.get(MainPageLinks.URL_CONTRIBUTORS_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/contributors",
      "time" : {
        "start" : 1728784125231,
        "stop" : 1728784125658,
        "duration" : 427
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55ec6566202a <unknown>\n#1 0x55ec653485e0 <unknown>\n#2 0x55ec653407eb <unknown>\n#3 0x55ec65330a09 <unknown>\n#4 0x55ec65332744 <unknown>\n#5 0x55ec65330cbd <unknown>\n#6 0x55ec6533052f <unknown>\n#7 0x55ec65330422 <unknown>\n#8 0x55ec6532e493 <unknown>\n#9 0x55ec6532eada <unknown>\n#10 0x55ec6534af41 <unknown>\n#11 0x55ec653dc8c5 <unknown>\n#12 0x55ec653bcb22 <unknown>\n#13 0x55ec653dbd7d <unknown>\n#14 0x55ec653bc8c3 <unknown>\n#15 0x55ec6538a6b3 <unknown>\n#16 0x55ec6538b68e <unknown>\n#17 0x55ec6562ca2b <unknown>\n#18 0x55ec656309b1 <unknown>\n#19 0x55ec65619225 <unknown>\n#20 0x55ec65631532 <unknown>\n#21 0x55ec655fe38f <unknown>\n#22 0x55ec65650f28 <unknown>\n#23 0x55ec656510f3 <unknown>\n#24 0x55ec65660e7c <unknown>\n#25 0x7f8c81894ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 28, in contributors_page_open\n    driver.get(MainPageLinks.URL_CONTRIBUTORS_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"8bca3ccebba11d3ebf0dc61fc7c53968\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728784125690,
      "stop" : 1728784125752,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "The Contributors Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "contributors_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestContributorsPage"
  }, {
    "name" : "host",
    "value" : "fv-az891-551"
  }, {
    "name" : "thread",
    "value" : "2367-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.contributors_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 4,
        "skipped" : 0,
        "passed" : 27,
        "unknown" : 0,
        "total" : 31
      },
      "items" : [ {
        "uid" : "461ecf1d8209d0fe",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/60//#testresult/461ecf1d8209d0fe",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55fe7a37102a <unknown>\n#1 0x55fe7a0575e0 <unknown>\n#2 0x55fe7a04f7eb <unknown>\n#3 0x55fe7a03fa09 <unknown>\n#4 0x55fe7a041744 <unknown>\n#5 0x55fe7a03fcbd <unknown>\n#6 0x55fe7a03f52f <unknown>\n#7 0x55fe7a03f422 <unknown>\n#8 0x55fe7a03d493 <unknown>\n#9 0x55fe7a03dada <unknown>\n#10 0x55fe7a059f41 <unknown>\n#11 0x55fe7a0eb8c5 <unknown>\n#12 0x55fe7a0cbb22 <unknown>\n#13 0x55fe7a0ead7d <unknown>\n#14 0x55fe7a0cb8c3 <unknown>\n#15 0x55fe7a0996b3 <unknown>\n#16 0x55fe7a09a68e <unknown>\n#17 0x55fe7a33ba2b <unknown>\n#18 0x55fe7a33f9b1 <unknown>\n#19 0x55fe7a328225 <unknown>\n#20 0x55fe7a340532 <unknown>\n#21 0x55fe7a30d38f <unknown>\n#22 0x55fe7a35ff28 <unknown>\n#23 0x55fe7a3600f3 <unknown>\n#24 0x55fe7a36fe7c <unknown>\n#25 0x7f8ffde94ac3 <unknown>",
        "time" : {
          "start" : 1728697238455,
          "stop" : 1728697238455,
          "duration" : 0
        }
      }, {
        "uid" : "f8468bfbf1a35956",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/f8468bfbf1a35956",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x56410337502a <unknown>\n#1 0x56410305b5e0 <unknown>\n#2 0x5641030537eb <unknown>\n#3 0x564103043a09 <unknown>\n#4 0x564103045744 <unknown>\n#5 0x564103043cbd <unknown>\n#6 0x56410304352f <unknown>\n#7 0x564103043422 <unknown>\n#8 0x564103041493 <unknown>\n#9 0x564103041ada <unknown>\n#10 0x56410305df41 <unknown>\n#11 0x5641030ef8c5 <unknown>\n#12 0x5641030cfb22 <unknown>\n#13 0x5641030eed7d <unknown>\n#14 0x5641030cf8c3 <unknown>\n#15 0x56410309d6b3 <unknown>\n#16 0x56410309e68e <unknown>\n#17 0x56410333fa2b <unknown>\n#18 0x5641033439b1 <unknown>\n#19 0x56410332c225 <unknown>\n#20 0x564103344532 <unknown>\n#21 0x56410331138f <unknown>\n#22 0x564103363f28 <unknown>\n#23 0x5641033640f3 <unknown>\n#24 0x564103373e7c <unknown>\n#25 0x7f2012e94ac3 <unknown>",
        "time" : {
          "start" : 1728610929796,
          "stop" : 1728610929796,
          "duration" : 0
        }
      }, {
        "uid" : "3679b5351cf66a88",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/3679b5351cf66a88",
        "status" : "passed",
        "time" : {
          "start" : 1728524628134,
          "stop" : 1728524628888,
          "duration" : 754
        }
      }, {
        "uid" : "b1e96ce173ecb808",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/b1e96ce173ecb808",
        "status" : "passed",
        "time" : {
          "start" : 1728438199967,
          "stop" : 1728438200725,
          "duration" : 758
        }
      }, {
        "uid" : "6261228362ea34a0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/6261228362ea34a0",
        "status" : "passed",
        "time" : {
          "start" : 1728351861682,
          "stop" : 1728351862521,
          "duration" : 839
        }
      }, {
        "uid" : "ab3b7e57ee1708ee",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/ab3b7e57ee1708ee",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.TimeoutException: Message: Can't find elements by locator ('xpath', '//section//a')",
        "time" : {
          "start" : 1728265774092,
          "stop" : 1728265794337,
          "duration" : 20245
        }
      }, {
        "uid" : "7e4309c023e04ebc",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/7e4309c023e04ebc",
        "status" : "passed",
        "time" : {
          "start" : 1728179417867,
          "stop" : 1728179418614,
          "duration" : 747
        }
      }, {
        "uid" : "9d5eeebded2b747",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/9d5eeebded2b747",
        "status" : "passed",
        "time" : {
          "start" : 1728092591661,
          "stop" : 1728092592405,
          "duration" : 744
        }
      }, {
        "uid" : "91a596d11e2d8b37",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/52//#testresult/91a596d11e2d8b37",
        "status" : "passed",
        "time" : {
          "start" : 1728006297011,
          "stop" : 1728006297757,
          "duration" : 746
        }
      }, {
        "uid" : "9b22e5689facde34",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/51//#testresult/9b22e5689facde34",
        "status" : "passed",
        "time" : {
          "start" : 1727919885668,
          "stop" : 1727919886466,
          "duration" : 798
        }
      }, {
        "uid" : "86624f110a7c9724",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/50//#testresult/86624f110a7c9724",
        "status" : "passed",
        "time" : {
          "start" : 1727833425066,
          "stop" : 1727833425782,
          "duration" : 716
        }
      }, {
        "uid" : "55b01055d07402dd",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/49//#testresult/55b01055d07402dd",
        "status" : "passed",
        "time" : {
          "start" : 1727747561778,
          "stop" : 1727747562498,
          "duration" : 720
        }
      }, {
        "uid" : "fde648ab788beaa5",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/48//#testresult/fde648ab788beaa5",
        "status" : "passed",
        "time" : {
          "start" : 1727660842885,
          "stop" : 1727660843624,
          "duration" : 739
        }
      }, {
        "uid" : "cdd0a00d8dbd3fe5",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/47//#testresult/cdd0a00d8dbd3fe5",
        "status" : "passed",
        "time" : {
          "start" : 1727574650046,
          "stop" : 1727574650789,
          "duration" : 743
        }
      }, {
        "uid" : "a826deba13a54d96",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/46//#testresult/a826deba13a54d96",
        "status" : "passed",
        "time" : {
          "start" : 1727487896404,
          "stop" : 1727487897142,
          "duration" : 738
        }
      }, {
        "uid" : "559feb60eaa277e0",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/45//#testresult/559feb60eaa277e0",
        "status" : "passed",
        "time" : {
          "start" : 1727401479954,
          "stop" : 1727401480714,
          "duration" : 760
        }
      }, {
        "uid" : "c4cb4dcef4eae3cf",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/44//#testresult/c4cb4dcef4eae3cf",
        "status" : "passed",
        "time" : {
          "start" : 1727315026114,
          "stop" : 1727315026865,
          "duration" : 751
        }
      }, {
        "uid" : "d8c7bb616f51508b",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/43//#testresult/d8c7bb616f51508b",
        "status" : "passed",
        "time" : {
          "start" : 1727228717875,
          "stop" : 1727228718573,
          "duration" : 698
        }
      }, {
        "uid" : "4a9db1a5db5c2ffd",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/42//#testresult/4a9db1a5db5c2ffd",
        "status" : "passed",
        "time" : {
          "start" : 1727142270633,
          "stop" : 1727142271384,
          "duration" : 751
        }
      }, {
        "uid" : "43321e49ce7340aa",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/41//#testresult/43321e49ce7340aa",
        "status" : "passed",
        "time" : {
          "start" : 1727055821972,
          "stop" : 1727055822692,
          "duration" : 720
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "bad55fc6c53e8aa4.json",
  "parameterValues" : [ ]
}